src/components/main-page/EventDetails/EventDetails.tsx

```diff
@@ -1,18 +1,24 @@
 import { Heading, Section, Text } from "components/ui";
-import useTranslation from "translations/hooks/useTranslations";
+import { useTranslations } from "next-intl";
 import styles from "./EventDetails.module.scss";
 
 export function EventDetails() {
-  const { t } = useTranslation();
+  const t = useTranslations();
 
   return (
     <Section id="event-details" className={styles.section}>
       <Heading className={styles.heading} withAccent>
         {t("eventDetails.title")}
       </Heading>
-      <Text className={styles.text}>{t("eventDetails.description1")}</Text>
-      <Text className={styles.text}>{t("eventDetails.description2")}</Text>
-      <Text className={styles.text}>{t("eventDetails.description3")}</Text>
+      <Text className={styles.text}>
+        {t.rich("eventDetails.description.main")}
```

Comment:
`t.rich` allows to handle custom tags in translations - in that case `<b>...</b>`